docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+
+Using these two resources, we can begin to get an idea of where to look for Silvio Caruso’s TEMP/TBLU files. We can see that the assets for Sapienza are in chunk26, and since that's a Hitman 1 location, chunk8 (Legacy) and chunk21 (Season1) may also have what we're looking for. Finally, chunk0 (Boot) and chunk1 (Base) are definitely worth checking as well, as these are essentially "main" chunks.
+
+Another hint: most (if not all) outfit TEMP/TBLU files start with the string `outfit_` typically followed by the level's name/codename or character's name, and end with the string `.pc_entitytemplate`.
+
+Therefore, we need to identify which `outfit_*****.pc_entitytemplate` file belongs to Silvio Caruso, and we know it's probably in either chunk0 (Boot), chunk1 (Base), chunk8 (Legacy), chunk21 (Season 1), or chunk26 (Sapienza).
+
+# Finding the Correct Files
+The [RPKG Tool](https://notex.app/rpkg/) allows us to search within the chunk files. Once setup, we can load the chunk files into the tool by using `Import` > `Import RPKGs folder` and pointing it to the `HITMAN 3\Runtime` folder. This will load all the chunk files into the tool, which is required when looking for the correct files. Next, we can search by using the left panel and going to `Search` > ` Search RPKGs`
+
+There are two ways to find the correct files. You can search for them yourself, or you can use community-made resources to quickly find the correct files. Using the community-made resources will save you time as most of the work has been done for you. Note that while this is easier, you will skip some valuable steps in the modding process, which might be useful to know if you decide to make more complex mods. Therefore, if you are creating your first Hitman mod, you are recommended to try and find the correct files yourself. 
+
+## Finding the Correct Files (Using Community Resources)
+Now, while we *could* search for the files manually, it isn't strictly necessary. Instead, we can use HMBM47's [Hitman 3 Outfits Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vRDiyiqdRebu0Olvvkr20CDhh6ANxu7FOQZ_O-1YHFN9e6kh0WmpbwDYbfgzevSvc3fO4_4Exu1fmQH/pubhtml#). We also have grappigegovert (who created the original outfit list) and 2kpr (who compiled a list of the NPCs by their names, their outfits, their pieces, etc.) to thank for this wonderful resource.
+
+If we access the `Outfits by Character Name` section of the spreadsheet, we can see each NPC's in-game name is located in the `Name` column on the right, while their outfit's filename is in the `Outfit` column. So, if we simply search for `Silvio Caruso`, we can see the filename we're looking for is `outfit_silviocarusso_actor_v0`.
+
+If we now go back to RPKG Tool and search for `outfit_silviocarusso_actor_v0`, the result should be several files with `outfit_silviocarusso_actor_vX.entitytemplate].pc_entityblueprint` in the name, in both `chunk1` and `chunk8`.
+
+## Finding the Correct Files (Manually)

```

Comment:
General for this section: adding a note about we may not have the hashes for what you're looking for (all files in general) would be a good point to have.